src/main/java/fi/fmi/avi/util/SubSolarPointUtils.java

```diff
@@ -0,0 +1,178 @@
+package fi.fmi.avi.util;
+
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Utility for calculating the sub-solar point (point on Earth where the Sun is directly overhead)
+ * based on the <a href="https://gml.noaa.gov/grad/solcalc/calcdetails.html">NOAA Solar Calculator</a> algorithm.
+ * <p>
+ * The Equation of Time is converted to longitude to yield an exact match to
+ * <a href="https://github.com/wmo-im/iwxxm/blob/v3.0-dev/IWXXM/examples/spacewx-A2-5.xml">WMO's IWXXM example</a>.
+ */
+public final class SubSolarPointUtils {
+
+    public static final double DAYLIGHT_SIDE_RADIUS_KM = 10_100.0;
```

Comment:
To avoid api changes when adding support for IWXXM 2025-2 space weather advisories, I'd suggest to name this constant to is suitable when we have `DAYSIDE` and `NIGHTSIDE` instead of just `DAYLIGHT_SIDE`.